dataset/dictionary/characters.json5

```diff
@@ -1868,9 +1868,9 @@
     notes: "Eleazar の原義はユダヤ人の男性名で、聖書にも Eleazar という司教が登場する。意味は「神が助けた」",
   },
   {
-    en: "Dori",
-    ja: "ドリー",
-    zhCN: "多莉",
+    en: "Dori Sangemah Bay",
+    ja: "ドリー・サングマハベイ",
+    zhCN: "多莉·桑歌玛哈巴依",
```

Comment:
@Bill-Haku I merged Dori and her last name Sangemah Bay as one word. Can you check if the Chinese is correct?

Reply:
That's correct.

Reply:
Thank you!